client/driver/docker_test.go

```diff
@@ -147,7 +172,7 @@ func TestDockerDriver_Start_Wait(t *testing.T) {
 }
 
 func TestDockerDriver_Start_Wait_AllocDir(t *testing.T) {
-	if !dockerLocated() {
+	if !dockerIsConnected() || dockerIsRemote() {
```

Comment:
This test fails with a remote docker host because the alloc dir cannot be mounted remotely.